cluster-operator/src/main/java/io/strimzi/operator/cluster/Main.java

```diff
@@ -240,7 +240,12 @@ private static KafkaConnectS2IAssemblyOperator createS2iOperator(Vertx vertx, Ku
             for (Map.Entry<String, String> clusterRole : clusterRoles.entrySet()) {
                 log.info("Creating cluster role {}", clusterRole.getKey());
 
-                try (BufferedReader br = new BufferedReader(new InputStreamReader(Main.class.getResourceAsStream("/cluster-roles/" + clusterRole.getValue()), Charset.defaultCharset()))) {
+                try (BufferedReader br = new BufferedReader(
+                        new InputStreamReader(
+                                Main.class.getResourceAsStream(
+                                        "/cluster-roles/"
+                                                + clusterRole.getValue()),
+                                Charset.defaultCharset()))) {
```

Comment:
Its very unlikely to be a problem, but this ends up being completely undefined behavior. Think about it: There's this file which gets edited on a developers machine (using who knows what char encoding), packaged into a jar, embedded in an image and then parsed at runtime using the default charset of the container. What ensures that the charset the developer's text editor or IDE used was the same as the default charset of the container? Do you even know what the default chartset of the container is, or that it doesn't very between nodes? I think it's much better to use UTF-8 here, and for all the YAMLs, that way at least you know for sure what you're reading the file as. (The other part of the problem is knowing what charset developers are using, but that's not really something you can solve in this PR).